ext/standard/tests/general_functions/gettype_basic.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Check for "unknown type"
+--CREDITS--
+Yulia Kostrikova <yuliakostrikova@gmail.com>
+# TestFest 2017 -  Kharkiv PHP UG 2017-10-28
+--FILE--
+<?php
+$fileName = dirname(__FILE__)."/test.txt";
+$r = fopen($fileName,'w');
+fclose($r);
+echo gettype($r);
+?>
+
+--CLEAN--
+<?php
+  $fileName = dirname(__FILE__)."/test.txt";
+  unlink($fileName);
+?>
+
+--EXPECTF--
```

Comment:
Nit:
```suggestion
--EXPECT--
```

Reply:
Hello, this one for example, I can also fix when I'll be merging it upstream. Let me know also please if you've spotted anything else...

Reply:
@petk Should be RTM after this quick fix.

Reply:
Awesome...